inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/AbstractSinkOperator.java

```diff
@@ -213,6 +217,23 @@ public void deleteOpt(StreamSinkEntity entity, String operator) {
         sinkFieldMapper.logicDeleteAll(entity.getId());
     }
 
+    @Override
+    public Map<String, String> parse2IdParams(StreamSinkEntity streamSink, List<String> fields) {
+        Map<String, String> param;
+        try {
+            param = objectMapper.readValue(streamSink.getExtParams(), Map.class);
+        } catch (Exception e) {
+            LOGGER.error("cannot parse properties of groupId={}, streamId={}, sinkName={}, the row properties is={}, "
+                            + "exception={}", streamSink.getInlongGroupId(), streamSink.getInlongStreamId(),
+                    streamSink.getSinkName(), streamSink.getExtParams(), e.getMessage());
+            return null;
```

Comment:
Did it need to return an empty Map?

Reply:
No, the error parsing will produce a **_null_** IdParams, which will be filtered to ensure correct config updating of other groupIds